sbt-dotty/src/dotty/tools/sbtplugin/DottyPlugin.scala

```diff
@@ -97,9 +97,8 @@ object DottyPlugin extends AutoPlugin {
       def delete(classes: Iterable[File]): Unit = {
         inherited.delete(classes flatMap { classFile =>
           val dottyFiles = if (classFile.getPath endsWith ".class") {
-            val f = new File(classFile.getAbsolutePath.stripSuffix(".class") + ".tasty")
-            if (f.exists) List(f)
-            else Nil
+            val prefix = classFile.getAbsolutePath.stripSuffix(".class")
+            List(".tasty", ".hasTasty").map(x => new File(prefix + x)).filter(_.exists)
```

Comment:
Documentation above this method needs to be updated to talk about .hasTasty files too.